src/zope/publisher/browser.py

```diff
@@ -676,6 +677,12 @@ def __init__(self, aFieldStorage):
             if hasattr(file, m):
                 d[m] = getattr(file, m)
 
+        if 'seekable' not in d and isinstance(
+            file, tempfile.SpooledTemporaryFile
+        ):
```

Comment:
```suggestion
        ): # Python 3.7 to 3.10
```

This comment will allow to remove this code once Python 3.10 support gets dropped in some years.